### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by yvlange**
Here is the content of my very first website!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  </head>
+  <body>
+    <header>
+      <h1>Welcome to my personal website</h1>
+      <nav>
+        <ul>
+          <li>Home</li>
+          <a href="#about">
+            <li>About Me</li>
+          </a>
```
Comment (mpagels) [inline index.html]: Would put the a href into the <li> element. See more here: https://developer.mozilla.org/de/docs/Web/HTML/Element/li

File: index.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+          <a href="#contact">
+            <li>Contact</li>
+          </a>
```
Comment (mpagels) [inline index.html]: Same.

**Review by mpagels — CHANGES_REQUESTED** [review]
Did leave some tiny things.
Very good job!

**Comment by hugh-burgess** [comment]
LGTM!

**Comment by johdyn** [comment]
LGTM

**Review by soerendolling — COMMENTED** [review]
lgtm

**Pull request merged**